packages/sdk/Dockerfile

```diff
@@ -169,5 +169,67 @@ ADD --chmod=644 \
 RUN tar -xJf /usr/share/cartesi-machine/images/linux-headers.tar.xz -C / && \
     rm /usr/share/cartesi-machine/images/linux-headers.tar.xz
 
+# Install migrate db cli
+ARG GO_MIGRATE_VERSION=4.18.2
+RUN <<EOF
+ARCH=$(dpkg --print-architecture || uname -m)
+case "$ARCH" in
+    amd64|x86_64)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-amd64.deb"
+        ;;
+    arm64|aarch64)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-arm64.deb"
+        ;;
+    armhf|armv7l)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-armv7.deb"
+        ;;
+    i386)
+        MIGRATE_URL="https://github.com/golang-migrate/migrate/releases/download/v${GO_MIGRATE_VERSION}/migrate.linux-386.deb"
+        ;;
+    *)
+        echo "Unsupported architecture: $ARCH"
+        exit 1
+        ;;
+esac
+
+curl -fsSL "$MIGRATE_URL" -o /tmp/migrate.deb
+dpkg -i /tmp/migrate.deb
+rm /tmp/migrate.deb
+migrate --version
+EOF
+
+
+# Get migration files for rollups-graphql
+ARG GRAPHQL_REPO=rollups-graphql
+ARG CARTESI_ROLLUPS_GRAPHQL_VERSION=2.3.8
+ARG GRAPHQL_GIT_FOLDER=dev/migrations
+ARG GRAPHQL_DEST_FOLDER=db/rollups-graphql
+
+RUN <<EOF
+mkdir -p "$GRAPHQL_DEST_FOLDER"
+curl -L "https://github.com/cartesi/$GRAPHQL_REPO/archive/refs/tags/v$CARTESI_ROLLUPS_GRAPHQL_VERSION.tar.gz" -o source.tar.gz
+echo "Downloaded: https://github.com/cartesi/$GRAPHQL_REPO/archive/refs/tags/v$CARTESI_ROLLUPS_GRAPHQL_VERSION.tar.gz"
+tar --strip-components=1 -xzf source.tar.gz "$GRAPHQL_REPO-$CARTESI_ROLLUPS_GRAPHQL_VERSION/$GRAPHQL_GIT_FOLDER" && \
+  mv "$GRAPHQL_GIT_FOLDER" "$GRAPHQL_DEST_FOLDER" && \
+  rm -f source.tar.gz
```

Comment:
Can´t we just extract what's necessary and send it directly to the target folder?

Something like that:

`curl ... | tar -xzf - db/miration/ -C /usr/share/cartesi/rollups-graphql/db `